### ISSUE TITLE
Wallet - Contacts - Not saving 

### ISSUE THREAD

**Issue by rafaroberto** [minima-global/Minima#266]
Hi,

Issue Impact: Minor
Device Manufacturer: Huawei 
Device Model: P20Pro 
Device OS: Android
Device OS Version: 10
Application Version: apk: v0.98.24 wallet version 1.1.2

What Happened?: I created and saved new contact amd despite a pop us saying the new contact was saved it did not appear on the list. I refreshed the page with no effect. I closed down the Wallet and reopened with no effect. No contact saved. 

What Did You Expect To Happen? To save new contact and for it to appear in the list immediately 

Steps to recreate:
1 Create contact
2.  Pop up saying contact saved
3. Refresh wallet 
4. No contract saved

I still have trouble adding screenshots from phone to GitHub. I'll get @eurobuddha to add in comments.

**Comment by rafaroberto** [comment]
Right. Sonit transpires I already had "Bob" in my contacts as "Boldor" not knowing that. So it looks like the Wallet knew the address was already saved wirh different contact but still popped up with a "Bob was saved to your contacts" even though it wasn't nor was the name changed.

Screenshots will shine additional light onto this 😁

**Comment by eurobuddha** [comment]
![raf6](https://user-images.githubusercontent.com/36900242/121491617-063c6780-c9ce-11eb-8174-3474be3991bf.jpg)
![raf5](https://user-images.githubusercontent.com/36900242/121491622-06d4fe00-c9ce-11eb-8b79-093d56c70348.jpg)

**Comment by eliasnemr** [comment]
This is now fixed in v1.1.3,  You'll now receive a message stating that the address already exists in your contacts if you try to re-add the same address.

**Comment by rafaroberto** [comment]
Thanks Elias,

I guess this can be closed I guess 😊

R

**Comment by eurobuddha** [comment]
we will close it once you have tasted 1.1.3 and tested it @rafaroberto

**Comment by rafaroberto** [comment]
Thanks!

**Comment by eliasnemr** [comment]
Bump
